src/adapter.ts

```diff
@@ -20,13 +20,12 @@ import {CasbinRule} from './casbinRule';
  * SequelizeAdapter represents the Sequelize adapter for policy storage.
  */
 export class SequelizeAdapter implements Adapter {
-    private connStr: string;
+    private param: string | Sequelize;
```

Comment:
Just add a new variable called ``sequelize`` instead of making ``param`` multi-type. Determine the param type in constructor. The internal variables should be only one-type to make the code easy for read.